app/src/test/contrib/actions/CoverageActionsTests.ts

```diff
@@ -0,0 +1,80 @@
+import { expect } from "chai";
+
+import { Sandbox } from "../../Sandbox";
+import { coverageActionCreators } from "../../../main/contrib/actions/coverageActionCreators";
+import { CoverageService } from "../../../main/contrib/services/CoverageService";
+import {Coverage, CoverageTypes} from "../../../main/contrib/actionTypes/CoverageTypes";
+import {createMockStore} from "../../mocks/mockStore";
+import {mockCoverageSet, mockDemographicDataset, mockScenario, mockTouchstone} from "../../mocks/mockModels";
+import {ScenarioAndCoverageSets} from "../../../main/shared/models/Generated";
+import {statePropsMapHelper} from "../../../main/contrib/helpers/statePropsMapHelper";
```

Comment:
I didn't catch this on the first PR, but I find this name rather unclear. Could you perhaps rename it to `mapStateToPropsHelper`?

Reply:
ok